tandemhunter-main/TandemHunter.py

```diff
@@ -0,0 +1,465 @@
+# Tandem Hunter: Identification of duplicated/amplified
+# regions based on the comparison
+# of normalise coverage between two intervals in given coverage file
+# (PER_TARGET_COVERAGE from Picard CollectHsMetrics).
+
+# Author: Helena L. Spiewak
+# Date created: 2021-04-06
+# Date modified: 2021-04-06
+
+# It is used to identify KMT2A/MLL partial tandem duplications and is based on
+# the method described by McKerrell et al., 2016, where they compare
+# the coverage of exon 3 and exon 27 of MLL/KMT2A.
+
+import pandas as pd
+import numpy as np
+import argparse
+import sys
+import os
+import json
+import multiprocessing as mp
+import fnmatch
+import functools
+from datetime import datetime
+import glob
+
+# define Picard CollectHsMetrics PER_TARGET_COVERAGE columns
+COV_COLUMN_DTYPES = {
+    "chrom": "category",
+    "start": "int",
+    "end": "int",
+    "length": "int",
+    "name": "category",
+    "%gc": "float64",
+    "mean_coverage": "float64",
+    "normalized_coverage": "float64",
+    "min_normalized_coverage": "float64",
+    "max_normalized_coverage": "float64",
+    "min_coverage": "int",
+    "max_coverage": "int",
+    "pct_0x": "float64",
+    "read_count": "int",
+}
+
+
+def tdh_argument_parser(args):
+    """ Parse arguments for Tandem Hunter. """
+    parser = argparse.ArgumentParser(
+        description=(
+            "Tandem Hunter: Identification of duplicated/amplified regions"
+            "based on comparing normalised coverage between two intervals"
+        )
+    )
+    required = parser.add_argument_group("required named arguments")
+    required.add_argument(
+        "-F", "--file", default=False, help="Path to coverage file OR"
+    )
+    required.add_argument(

```

Comment:
You may be able to enforce not using -F and -B together up-front, by creating them as a 'mutually exclusive group' or as subparsers. See example threads here: 
https://stackoverflow.com/questions/39092149/argparse-how-to-make-mutually-exclusive-arguments-optional 
https://stackoverflow.com/questions/7869345/how-to-make-python-argparse-mutually-exclusive-group-arguments-without-prefix